prime-router/src/main/kotlin/cli/tests/LoadTestSimulator.kt

```diff
@@ -250,6 +250,18 @@ abstract class LoadTestSimulator : CoolTest() {
             stracSender,
             true
         )
+        val twoThreadsX1000with50msDelay = Simulation(
```

Comment:
The 50ms delay is the main change here - this tames the test down a bit, and is a bit more representative of the random hits we get in Prod.